### ISSUE TITLE
macOS sound improvement

### ISSUE THREAD

**Issue by jjanyan** [franciscolourenco/done#135]
I wanted to suggest a change the macOS sound notification to use the applescript notification instead of the terminal bell. It could also be upgraded to support different sounds.

```fish
            else if type -q osascript # AppleScript
                if test "$__done_notify_sound" -eq 1
                    osascript -e "display notification \"$message\" with title \"$title\" sound name \"Glass\""
                else
                    osascript -e "display notification \"$message\" with title \"$title\""
                end
```

I'm happy to send a PR over, but before I do, is this something you'd be interested in including?

**Comment by smiller171** [comment]
Unfortunately it looks like @franciscolourenco isn't maintaining this anymore

**Comment by franciscolourenco** [comment]
I've not been spending much time with this project lately, but I will probably have more time moving forward. 
@jjanyan what you suggest makes sense, feel free to open a PR. Supporting multiple sounds however might be overkill, no?

**Comment by jjanyan** [comment]
PR created https://github.com/franciscolourenco/done/pull/136

**Comment by franciscolourenco** [comment]
Merged, thanks!. It will be released in v1.17.0
